frontend/src/credit_transfers/components/CreditTransferFormDetails.js

```diff
@@ -95,8 +96,8 @@ class CreditTransferFormDetails extends Component {
             />
           </div>
           <span>per credit for a total value of </span>
-          <span>{numeral(this.props.totalValue).format(NumberFormat.CURRENCY)}</span>
-          <span> effective on Director&apos;s approval</span>
+          <span>{numeral(this.props.totalValue).format(NumberFormat.CURRENCY)} CAD.</span>
+          {/* <span> effective on Director&apos;s approval</span> */}
```

Comment:
Does this still need to be here for future reference?

Reply:
i removed it